Makefile

```diff
@@ -30,6 +30,8 @@ battery-scripts:
 keyboard-scripts:
 	@echo "Installing keyboard scripts"
 	cp keyboard/keyboard_toggle ${TARGET}
+	cp keyboard/lightning/lightbar_toggle ${TARGET}
+	cp keyboard/lightning/intensity ${TARGET}
```

Comment:
I think it would be better to have a clearer name for this like `keyboard_intensity` since it'll come in the `.local/bin`